### PR TITLE
feat (Codespaces) - Added Commonly used tasks to `tasks.json` for VSCode users. 

### PR DESCRIPTION
**Pull request by ShreyasJejurkar**
Further, improving codespaces/VScode experience for users.

1. Added EditorConfig as a preinstalled extension. 
2. Added commonly performed tasks to the `tasks.json` file. So that one can invoke that task through VSCode directly by pressing `CTRL + SHIFT + P` and then select option `Run Tasks` and then actual tasks. (We can add more tasks going forward.)

Press `CTRL + SHIFT + P` and search for Run Tasks. Select option like below. 

image

Choose the task from list and hit enter to execute. 

image

Some tasks may have more options to select, for example in the below case while building an entire repository, one can select different configurations like Debug OR Release. And based on the selection the command will run. 

image

And given task will start running in the terminal. 

image

### CODE REVIEW

**Review by captainsafia — APPROVED** [review]
Looks good overall! Left a few notes inline.

Another set of tasks that might be good to add is build/test for a particular project. I wonder if there is a way to craft a task input that lists all the .csproj files in the repo...

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -19,12 +19,16 @@
...
+		"EditorConfig.EditorConfig",
+    "k--kato.docomment"
 	],
 
 	"settings": {
 		// Loading projects on demand is better for larger codebases
-		"omnisharp.enableMsBuildLoadProjectsOnDemand": true
+		"omnisharp.enableMsBuildLoadProjectsOnDemand": true,
+    "omnisharp.enableRoslynAnalyzers": true,
+    "omnisharp.enableEditorConfigSupport": true,
```
[the remark this answers is not in the document]
Comment (ShreyasJejurkar) [reply .devcontainer/devcontainer.json]: Formatted with VSCode formatter for JSON!

**Comment by ShreyasJejurkar** [comment]
Hi @captainsafia, How do you edit the ResX file in VSCode? 
1. Raw XML editing.
2. Any extension you use?

If you use any extension for the editing, let me know we will add those to recommendations as well in docs and in `devcontainer.json` for streamline experience.

**Review by captainsafia — APPROVED** [review]

**Comment by captainsafia** [comment]
>  How do you edit the ResX file in VSCode?

I haven't figured out a great workflow for this yet unfortunately. On the rare occasions where I do interact with ResX files, I usually end up editing the raw XML. :/

I'm not sure if there is a CLI tool for generating code from a ResX. That would help with at least part of the process.

**Pull request merged**